### ISSUE TITLE
TableSize TRIM built-in function error

### ISSUE THREAD

**Issue by tdinh2011** [trimble-oss/dba-dash#868]
DBADash is reporting errors for the new TableSize collection. In my case, it seems to affect only SQL Server 2016.

```
Microsoft.Data.SqlClient.SqlException (0x80131904): 'TRIM' is not a recognized built-in function name.
'TRIM' is not a recognized built-in function name.
```

**Comment by DavidWiseman** [comment]
Thanks for reporting this.  I'll get it fixed soon.

**Comment by DavidWiseman** [comment]
The issue is fixed in 3.4.2 🎉
